server/routes/pollRouter.js

```diff
@@ -0,0 +1,45 @@
+const router = require('express').Router();
+const { cloudinary } = require('../utils/cloudinary');
+const Poll = require('../models/pollModel');
+
+router.post('/upload', async (req, res) => {
+    try {
+        let { imageOne, imageTwo, question, friendlist, userId } = req.body;
+
+        if (!question || !friendlist) {
+            return res
+                .status(400)
+                .json({ msg: 'Not all required fields have been entered.' });
+        }
+        if (!imageOne || !imageTwo) {
+            return res
+                .status(400)
+                .json({ msg: 'Both images are required.' });
+        }
+
+        const uploadedResponseOne = await cloudinary.uploader.upload(imageOne, {
```

Comment:
We can probably put the uploading code into a reusable function, and just take in the image (`imageOne` in this case) as a parameter. 

Not a bit deal here but I can forsee this being helpful just to isolate cloudinary specific code